### ISSUE TITLE
(AngularJS)Aria-setsize attribute missing for the default values in the MultiSelect

### ISSUE THREAD

**Issue by PekoPPT** [telerik/kendo-ui-core#5132]
### Bug report

The "aria-setsize" attribute is missing when initializing a MultiSelect with default selected values in AngularJS

### Reproduction of the problem
1. Open [this Dojo](https://dojo.telerik.com/OGulIsEk) and run it
2. Right-click on one of the initially selected items in the MultiSelect. 
3. The selected DOM element will be a "li" tag
4. Select a new item in the MultiSelect and watch how the definition of the "li" tag will change
5. When selecting a third element in the MultiSelect. All the three "li" tags will receive an **aria-setsize="3"** tag, which wasn't presented for the default selected values 

### Workdaround
1. Set the **valueprimitive** configuration to **false** or comment it
2. Define the initially selected values like this:
`$scope.selectedIds = [ {ProductID: 4, ProductName: "Chef Anton's Cajun Seasoning"}, {ProductID: 7, ProductName: "Uncle Bob's Organic Dried Pears"}  ];`

[Here is a Dojo ](https://dojo.telerik.com/aFOLOTUn)demonstrating the workaround.

### Current behavior
When initializing a MultiSelect with default values, the **aria-setsize** attribute is missing for these values. 
![aria-size](https://user-images.githubusercontent.com/41293735/60034781-fc6fab80-96b3-11e9-9a4a-bd629effefac.png)

### Expected/desired behavior
The default MultiSelect values should have an **aria-setsize** attribute added to their selected values
![aria-size1](https://user-images.githubusercontent.com/41293735/60034885-3b9dfc80-96b4-11e9-80f1-81b8941f6bc2.png)
 
### Environment

* **Kendo UI version:** 2019.2.619
* **jQuery version:** x.y
* **Browser:** [all]

**Comment by Dimitar-Goshev** [comment]
Closing this issue with Won't fix - [Kendo UI and AngularJS 1.x Support End of Life](https://www.telerik.com/blogs/kendo-ui-angularjs-1.x-support-end-life-plans) - R2 2022.
